index.html

```diff
@@ -57,14 +70,44 @@
         }
 
         .description {
-            font-weight: 500;
+            font-weight: 400;
+            font-size: 1.3rem;
             margin-bottom: 5px;
+            color: #424242;
         }
 
         .summary {
             margin-top: 20px;
             text-align: center;
         }
+
+        h1 {
+            font-size: 2rem;
+            font-weight: 400;
+            color: #424242;
+        }
+
+        .btn {
+            background-color: #26a69a;
+        }
+
+        .btn:hover {
+            background-color: #2bbbad;
+        }
+
+        .slider {
+            width: 100%;
+        }
```

Comment:
Added a width: 100%; style rule to the .slider class for better alignment with other elements in the scoring section.